src/app/integrations/edit-page/save-or-add-step/save-or-add-step.component.ts

```diff
@@ -79,6 +79,20 @@ export class IntegrationsSaveOrAddStepComponent extends FlowPage implements OnIn
     }
   }
 
+  addNew(type: string) {
+    this.currentFlow.events.emit({
+      kind: 'integration-add-step',
+      type: type,
+    });
+  }
+
+  showPopouts(type: string) {
```

Comment:
How are we handling _which_ popover to show? Or does it not matter because only one will ever be present? We need to test manually for any edge cases.